### PR TITLE
HollowNode refactoring

### PR DESCRIPTION
**Pull request by gmarek**

### CODE REVIEW

**Comment by gmarek** [comment]
@k8s-bot test this please (flake)

File: pkg/kubemark/hollow-kubelet.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	"k8s.io/kubernetes/pkg/volume/empty_dir"
+
+	"github.com/golang/glog"
+)
+
+type HollowKubelet struct {
+	KubeletConfig *kubeletapp.KubeletConfig
+}
+
+func makeTempDirOrDie(prefix string, baseDir string) string {
```
Comment (wojtek-t) [inline pkg/kubemark/hollow-kubelet.go]: This function has nothing to do with kubemark (in fact there is exactly the same function in cmd/integration/integration.go)
Can you please refactor it by moving this function (probably) to test/integration/utils.go

**Comment by wojtek-t** [comment]
LGTM - just one minor comment

**Comment by gmarek** [comment]
Done. PTAL

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**